fastly/compute_package.go

```diff
@@ -30,6 +30,7 @@ type PackageMetadata struct {
 	Language    *string  `mapstructure:"language"`
 	Name        *string  `mapstructure:"name"`
 	Size        *int64   `mapstructure:"size"`
+	ClonedFrom  *string  `mapstructure:"cloned_from"`
```

Comment:
Can we sort the fields alphabetically (for code base consistency as well as making it easier for users to search for specific fields).

Reply:
It's not necessary I guess, but you could always regenerate the fixtures and update the tests to check for the existence of this field.

Reply:
Done and done!